flexget/plugins/list/radarr_list.py

```diff
@@ -0,0 +1,593 @@
+            # or try to parse the 'quality' entry somehow?
+            quality_profile_id = 1
+
+            try:
+                add_result = self.service.add_movie(
+                    result["title"],
+                    quality_profile_id,
+                    result["titleSlug"],
+                    result["images"],
+                    result["tmdbId"],
+                    root_folder_path,
+                )
+                log.verbose("Added movie %ls to Radarr list", result["title"])
+            except RadarrMovieAlreadyExistsError:
+                log.warning(
+                    "Could not add movie %ls because it already exists on Radarr", result["title"]
+                )
+            except RadarrRequestError as ex:
+                log.error("The movie add command raised exception: %s", ex)
+        else:
+            log.verbose(
+                "The lookup for entry %s did not return any results."
+                "Can not add the movie in Radarr.",
+                entry,
+            )
+
+    def _from_iterable(self, it):
+        # The following implementation is what's done in every other
+        # list plugin. Does not makes sense.
+        # As I understand it, it aims to be an override to base class Set._from_iterable.
+        # However, thats a classmethod and this does not even match its signature
+        # https://blog.devzero.com/2013/01/28/how-to-override-a-class-method-in-python/
+        return set(it)
+
+    def get(self, entry):
+        # Here we must return a matching entry using the provided the "hint" entry
+        # In Radarr's case we will check different movie ids and then resort to name/year.
+        return self._find_matching_entry(entry)
+
+    @property
+    def items(self):
+        """ Property that returns all items and only loads them all items when needed """
+        if self._movie_entries is None:
+            self._movie_entries = self._get_movie_entries()
+        return self._movie_entries
+
+    @property
+    def immutable(self):
+        # Here we return True if it's not possible to modify the list.
+        # Could depends on the configuration (self.config).
+        # But as long it's only about movies it should be alright
+        # for Radarr
+        return False
+
+    @property
+    def online(self):
+        # Radarr is an online service, so yes...
+        return True
+
+    def _find_matching_entry(self, entry):
+        """
+        Finds a movie by first checking agains the ids of the provided entry,
+        and if none matches, check by title name
+        """
+        for movie_entry in self.items:
+            # First check if any of the id attributes match
+            for id_attribute in ["tmdb_id", "imdb_id", "radarr_id"]:
+                if id_attribute in entry and id_attribute in movie_entry:
+                    if entry[id_attribute] == movie_entry[id_attribute]:
+                        # Perfect match!
+                        return movie_entry
```

Comment:
A lot of indentation here.
```suggestion
            for id_attribute in ["tmdb_id", "imdb_id", "radarr_id"]:
                if id_attribute in entry and entry.get(id_attribute) and movie_entry.get(id_attribute):
                    # Perfect match!
                    return movie_entry```